core/probe.h

```diff
@@ -56,6 +56,14 @@ class BenchSuite;
 class Server;
 class ToolManager;
 
+/**
```

Comment:
There's no tight coupling between Probe lifetime & QCoreApplication lifetime before (because of the deleteLater), thus this comment does not make sense without the code changes.